cmd/go2proto/testdata/go2proto.to.go

```diff
@@ -81,6 +139,31 @@ func (x *Root) ToProto() *destpb.Root {
 	}
 }
 
+func RootFromProto(v *destpb.Root) *Root {
+	if v == nil {
+		return nil
+	}
+	f12 := &url.URL{}
+	f12.UnmarshalBinary(v.Url)
+	f13 := &key.Deployment{}
+	f13.UnmarshalText([]byte(v.Key))
```

Comment:
I don't think we should really be ignoring these errors :\ ... we're losing a lot of safety here.